assets/src/domain/eventEditor/ui/datetimes/datesList/List.tsx

```diff
@@ -32,8 +33,48 @@ const List: React.FC<ListProps> = ({ datetimes }) => {
 		</>
 	);
 
+	const data = [
+		{
+			key: '1',
+			name: 'John Brown',
+			age: 32,
+			address: 'New York No. 1 Lake Park',
+		},
+		{
+			key: '2',
+			name: 'Jim Green',
+			age: 42,
+			address: 'London No. 1 Lake Park',
+		},
+		{
+			key: '3',
+			name: 'Joe Black',
+			age: 32,
+			address: 'Sidney No. 1 Lake Park',
+		},
+	];
+
+	const columns = [
+		{
+			title: 'Name',
+			dataIndex: 'name',
+			key: 'name',
+		},
+		{
+			title: 'Age',
+			dataIndex: 'age',
+			key: 'age',
+		},
+		{
+			title: 'Address',
+			dataIndex: 'address',
+			key: 'address',
+		},
+	];
+
 	return (
 		<div>
+			<EspressoDragSortingTable columns={columns} dataSource={data} />
```

Comment:
were you still going to remove this?

Reply:
cleaned up: https://github.com/eventespresso/event-espresso-core/pull/2365/commits/f9b70ef7ba67323e026637da77a8d471e45d7805